.github/workflows/bootstrap_region.yml

```diff
@@ -34,14 +34,24 @@ permissions:
   contents: read
 
 jobs:
-  cdk:
-    name: Install CDK
+  bootstrap:
+    name: Bootstrap Region
     runs-on: ubuntu-latest
     permissions:
       contents: write
```

Comment:
Please correct if I am wrong, but you are not pushing commits or creating PR in this action, why do you need `contents: write` here? It will lower openssf scorecard.

Reply:
Good point - @sthulb can we remove it?

Reply:
Removed in both